src/Report.php

```diff
@@ -23,6 +23,13 @@ class Report
      */
     protected $config;
 
+    /**
+     * The original error.
+     *
+     * @var \Throwable|array|null
+     */
+    protected $original;
```

Comment:
Apologies, in our discussion I meant the field to be `originalError` and the standard getter `getOriginalError`.

```suggestion
    protected $originalError;
```